mipengine/node/monetdb_interface/views.py

```diff
@@ -19,7 +19,7 @@ def create_view(
     table_name: str,
     columns: List[str],
     filters: dict,
-    enable_min_rows_threshold=False,
+    check_min_rows=False,
```

Comment:
Nit: Choosing a name like `check_min_rows` for a method exposed in the algorithm flow (whether this name is good or bad) doesn't mean that we should change the API of methods in completely unrelated scopes. 
Maybe it makes sense in some scope and not in others.

Reply:
I agree with your point and have done exactly that in `create_primary_data_views` and `create_data_modeL_views`.

For the `check_min_rows`, I think, it's the same thing in both contexts, that's why I changed it to be the same, not to confuse people, since it also looks better. What do you think?

Reply:
I get your point, it's less confusing. 

I was just thinking that in the algorithm context it is something that will be typed often, so very large names are not great. On the other hand, in the MonetDB interface it is better to be more explicit, as the name might not be very self-explanatory. 

But it was just a remark. It's your call.